ledger/acctupdates_test.go

```diff
@@ -1343,3 +1392,118 @@ func TestCompactDeltas(t *testing.T) {
 	require.Equal(t, 1, outCreatableDeltas[101].ndeltas)
 
 }
+
+func TestReproducibleCatchpointLabels(t *testing.T) {
+	if runtime.GOARCH == "arm" || runtime.GOARCH == "arm64" {
+		t.Skip("This test is too slow on ARM and causes travis builds to time out")
+	}
+	proto := config.Consensus[protocol.ConsensusCurrentVersion]
+
+	ml := makeMockLedgerForTracker(t, false)
+	defer ml.close()
+	ml.blocks = randomInitChain(protocol.ConsensusCurrentVersion, 1)
+	ml.deltas = []StateDelta{StateDelta{}}
+
+	accts := []map[basics.Address]basics.AccountData{randomAccounts(20, true)}
+	rewardsLevels := []uint64{0}
+
+	pooldata := basics.AccountData{}
+	pooldata.MicroAlgos.Raw = 100 * 1000 * 1000 * 1000 * 1000
+	pooldata.Status = basics.NotParticipating
+	accts[0][testPoolAddr] = pooldata
+
+	sinkdata := basics.AccountData{}
+	sinkdata.MicroAlgos.Raw = 1000 * 1000 * 1000 * 1000
+	sinkdata.Status = basics.NotParticipating
+	accts[0][testSinkAddr] = sinkdata
+
+	au := &accountUpdates{}
+	cfg := config.GetDefaultLocal()
+	cfg.CatchpointInterval = 500
```

Comment:
looks like setting `CatchpointInterval=20` would speedup the test with the same level of reproducibility

Reply:
It needs to be greater than the lookback, which defaults to 320.
And yes, we can use a custom protocol here yo do that.

Reply:
done